content/v2/openapi.yml

```diff
@@ -3656,6 +3689,25 @@ components:
           -----END RSA PRIVATE KEY-----
     Contact:
       type: object
+      required:
+        - id
+        - account_id
+        - label
+        - first_name
+        - last_name
+        - organization_name
```

Comment:
Note that organization name, job title, fax and address2 are required. So while the key will exist it's value will be null.

Reply:
I thought so too at first, but it appears we actually return empty strings and not null values in case of missing attributes:

```
      {
         "account_id" : 132831,
         "address1" : "Test St.",
         "address2" : "",
         "city" : "Test City",
         "country" : "US",
         "created_at" : "2022-11-23T08:49:19Z",
         "email" : "test@test.com",
         "fax" : "",
         "first_name" : "T",
         "id" : 54567,
         "job_title" : "",
         "label" : "Test",
         "last_name" : "Est",
         "organization_name" : "",
         "phone" : "+639271234567",
         "postal_code" : "1234",
         "state_province" : "Test",
         "updated_at" : "2022-11-23T08:49:19Z"
      }
```